File: aoj/volume0/n0045/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int totalPrice = 0;
        int totalQuantity = 0;
        int count = 0;

        while (sc.hasNext()) {

            String[] input = sc.next().split(",");

            int price = Integer.parseInt(input[0]);
            int quantity = Integer.parseInt(input[1]);

            totalPrice += price * quantity;
            totalQuantity += quantity;

            count++;

        }

        System.out.println(totalPrice);
        System.out.println(Math.round((double) totalQuantity / count));

    }

}

File: aoj/volume0/n0045/Main.cpp
#include <iostream>

using namespace std;

int main() {

    int totalPrice = 0;
    int totalQuantity = 0;
    int count = 0;

    int price, quantity;
    char comma;

    while (cin >> price >> comma >> quantity) {

        totalPrice += price * quantity;
        totalQuantity += quantity;

        count++;

    }

    int averageQuantity = (double) totalQuantity / count + 0.5;

    cout << totalPrice << endl;
    cout << averageQuantity << endl;

}
